src/plugins/tool_extrude.py

```diff
@@ -29,128 +29,102 @@ def __init__(self, api):
             "Extrude", None)
         self.action.setStatusTip("Extude region")
         self.action.setCheckable(True)
-        self.action.setShortcut(QtGui.QKeySequence("Ctrl+9"))
+        self.action.setShortcut(QtGui.QKeySequence("Ctrl+0"))
```

Comment:
It's because it's already the shortcut for resetting camera position. I think I will change the camera one to something else.